presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/ImagesView.java

```diff
@@ -30,6 +38,12 @@ public void callServiceBtnPressed() {
     }
 
     public void showError() {
+        tvlabel.setEnabled(true);
         tvlabel.setText(R.string.connection_error);
     }
+
+    public void showImagesInCardView(List<ImageEntity> images) {
+        recyclerView.setAdapter(new RecyclerViewAdapter(this.getContext(), images));
```

Comment:
check if getContext is null, and separate in a local variable

Reply:
ok done